leveler/image_generators.py

```diff
@@ -585,6 +593,12 @@ def _write_unicode(text, init_x, y, font, unicode_font, fill):
             fill=rep_fill,
         )  # Exp Text
 
+        credits_name = (
+            credits_name.upper()
+            if label_fnt.getmask(credits_name.upper()).getbbox()[2]
+            <= label_fnt.getmask("BALANCE").getbbox()[2]
+            else "BALANCE"
```

Comment:
I changed it to a variable of constant size for now, although I'm a bit wary of this in case further profile customization is added in the future (i.e., custom fonts)